src/compiler/generate-xspec-tests.xsl

```diff
@@ -57,6 +57,8 @@
     <namespace-alias stylesheet-prefix="__x" result-prefix="xsl" />
     <variable name="x:stylesheet-uri" as="xs:string" select="'{$stylesheet-uri}'" />
   	<output name="x:report" method="xml" indent="yes" />
+    <!-- issue #93 -->
+    <variable name="x:compiled-xsl-uri" select="static-base-uri()" as="xs:anyURI"/>
```

Comment:
Assuming the variable is private, isn't the namespace prefix `impl` more suitable than `x`?